lib/XML/LibXML/Document.pm

```diff
@@ -45,7 +46,10 @@ method doctype is aka<type> {
     #~ This read-only property is an object that implements the DOMImplementation interface.
 
 #| This read-only property is an object that implements the Element interface.
-method documentElement is aka<root> {
+method documentElement 
```

Comment:
What do you think of these aka-methods at all? More and more I think we should stick to the longer more xmllish names...